### PR TITLE
metadata: Add uncompressed-sha256

### PR DESCRIPTION
**Pull request by cgwalters**
See https://github.com/coreos/stream-metadata-go/pull/11

### CODE REVIEW

File: metadata/stream/rationale.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ architectures:
               location: https://artifacts.example.com/dsB2fnzP7KhqzQ5a.vmdk.xz
               signature: https://artifacts.example.com/dsB2fnzP7KhqzQ5a.vmdk.xz.sig
               sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
-              # Could also include artifact size/uncompressed-size/uncompressed-sha256 from meta.json
+              uncompressed-sha256: 38acb15d02d5ac0f2a2789602e9df950c380d2799b4bdb59394e4eeabdd3a662
```
Comment (jlebon) [inline metadata/stream/rationale.yaml:25]: Can we also copy/paste this to the other artifacts so it's clear it's not just an AWS thing?

File: metadata/stream/sample.json [hunk 2]
```diff
@@ -56,7 +56,8 @@
               "disk": {
                 "location": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20190905.0/x86_64/fedora-coreos-30.20190905.0-openstack.qcow2.xz",
                 "signature": "https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20190905.0/x86_64/fedora-coreos-30.20190905.0-openstack.qcow2.xz.sig",
-                "sha256": "7b6608f03bcf98f41494c0a71fa518256798065c2516ff757e6bdd766f870ede"
+                "sha256": "7b6608f03bcf98f41494c0a71fa518256798065c2516ff757e6bdd766f870ede",
+                "uncompressed-sha256": "38acb15d02d5ac0f2a2789602e9df950c380d2799b4bdb59394e4eeabdd3a662"
```
Comment (jlebon) [inline metadata/stream/sample.json:60]: Here too, but meh we can just refresh that file from real output once we have it to make it easier.

**Comment by bgilbert** [comment]
Since we're now shipping stream metadata including this field, it'd be good to get these docs landed.

File: metadata/stream/sample.json (hunk 2 above)
Comment (jlebon) [reply metadata/stream/sample.json:60]: Should be able to copy-paste from https://builds.coreos.fedoraproject.org/streams/stable.json now!

**Comment by travier** [comment]
Gentle ping for @cgwalters maybe?

File: metadata/stream/rationale.yaml (hunk 1 above)
Comment (bgilbert) [reply metadata/stream/rationale.yaml:25]: @cgwalters Looks like this still needs doing?

**Comment by cgwalters** [comment]
Updated :new:

**Review by bgilbert — APPROVED** [review]
There are some nits, but for convenience I'll just approve this and submit the fixes as a followup.

**Pull request merged**